protocol/bin/flow.js

```diff
@@ -197,7 +163,7 @@ function parseMaybe (t) {
 }
 
 function parseUnion (unionTypes) {
-  return unionTypes.map(parseInnerType).join(' | ')
+  return '\n    ' + unionTypes.map(parseInnerType).join('\n  | ')
```

Comment:
Does this work that way? maybe we should use the new format with a leading `|` https://github.com/facebook/flow/blob/master/Changelog.md